server/server.go

```diff
@@ -32,6 +36,43 @@ func (r *rodeServer) BatchCreateOccurrences(ctx context.Context, occurrenceReque
 	}, nil
 }
 
+func (r *rodeServer) AttestPolicy(ctx context.Context, request *pb.AttestPolicyRequest) (*pb.AttestPolicyResponse, error) {
+	log := r.logger.Named("AttestPolicy")
+	log.Debug("received requests")
+
+	// check OPA policy has been loaded
+
+	// fetch occurrences from grafeas
+	listOccurrencesResponse, err := r.grafeasClient.ListOccurrences(ctx, &grafeas.ListOccurrencesRequest{Filter: fmt.Sprintf("resourceUri:%s", request.ResourceURI)})
```

Comment:
not sure if this filter string is correct.  thought it'd be something like `resource.uri = %s`